docs/architecture/ADR-009-liquidity-mining

```diff
@@ -0,0 +1,176 @@
+# ADR 009: Liquidity Mining
+
+## Changelog
+
+- May 1 2022: Initial Draft (@toteki)
+
+## Status
+
+Proposed
+
+## Context
+
+Umee wishes to add support for liquidity mining incentives; i.e. additional rewards on top of the normal `x/leverage` lending APY for supplying base assets.
+
+The following is a record of preferences and/or requirements collected during initial exploration:
+
+| Q | A |
+| - | - |
+| Where do the reward tokens come from? | The community pool (which can be funded) but at a fixed initial amount. |
+| Are rewards liquid? | Yes. |
+| How do users participate? | Lockup uTokens from lending. |
+| What are the lockup periods? | Tiers: 24h/14d/30d. Three tiers typically provide a clean UX. Periods would be difficult to change post-launch |
+| Are users required to participate in liquidity mining? | No. (Regular lending and usage as collateral are possible without locking) |
+| When are rewards distributed? | Every epoch, e.g. 24h. |
+| How are interest rates calculated? | Fixed total rewards: A flat total amount of incentive is to be divided proportionately across lenders of a given token type (e.g. Atom) weighted by lock duration. Effective interest rate depends on (flat reward amount / TVL). |
+| Can you use locked uTokens as collateral to borrow? | Yes. As a consequence, locked tokens must be able to be liquidated. |
+| Are external incentives enabled and matched? | Funds are first distributed/supplied to the community pool (both native and IBC-based assets) and then they’re distributed to the incentive module via the governance proposal. Incentive token denomination will not affect implementation. |
+| Will mining exist over a fixed period of time or not? | 1 Gov Proposal => Flat Total Reward Amount over Fixed Duration (Simple implementation but higher gov burden): Individual votes periodically refresh it (e.g. vote Y/N add +4M umee to be distributed proportionally to Atom lenders from StartDate to EndDate - must vote again to continue after EndDate). |
+
+## Decision
+
+The general approach will be to create an `x/incentive` module with a small surface of interaction with `x/leverage`.
```

Comment:
are we going to use Osmosis incentive module or the one from gravity dex? Sounds like this design is heavily influenced by Osmosis incentives module.

Reply:
I think it will be better to collaborate in notion about the final design, it's better to share comments there and edit than in ADR.  Let's clean up this ADRfill as much as we can, and merge as _Proposed_ and continue with the design in Notion.